src/fmu/dataio/dataio.py

```diff
@@ -853,7 +854,7 @@ def export(self, obj, return_symlink=False, **kwargs) -> str:
             export_metadata_file(metafile, metadata, savefmt=self.meta_format)
             logger.info("Metadata file is: %s", metafile)
         else:
-            warnings.warn("Metadata are invalid and will not be exported!", UserWarning)
+            warnings.warn("Data will be exported, but without metadata.", UserWarning)
```

Comment:
Possible to specify reason (which part is missing) in the warning?

Reply:
Yes it is there already from a previous commit: https://github.com/equinor/fmu-dataio/blob/2ea864740e8f98d39162612457fe05ce4d8f9ea8/src/fmu/dataio/dataio.py#L102 and below